demo.py

```diff
@@ -51,7 +51,7 @@ def train_model(X_train, y_train, X_test, y_test, lmd):
     TREE_DEPTH = 3
 
     # define sampler
-    dwave_sampler = DWaveSampler(solver={'qpu': True})
+    dwave_sampler = DWaveSampler(solver=dict(topology__type='pegasus', qpu=True))
```

Comment:
I would leave this as it was.

We don't want to _always_ run on Pegasus, but to make sure it _can_ run on Pegasus.